python/create_large_test_files.py

```diff
@@ -37,8 +37,10 @@ class that creates test files with a new maximum number of rows. The class is
 
 #The class is instantiated with an existing source file. 
 large_file = LargeTestFiles(
-	source_filename="../edits_files/clean_file_1000_rows.txt", 
-	output_filename="../edits_files/clean_file_4000_rows.txt" )
+	source_filepath="../edits_files/",
+	source_filename= "clean_file_100_rows.txt",
+	output_filepath = "../edits_files/",
+	output_filename="clean_file_4000_rows.txt" )
```

Comment:
These changes look good. Is there a way we can integrate them with the current .yaml file for configuration? Or add a second .yaml file?

Reply:
So would you envision adding in the .yaml file a configuration for where large test files would go - I think that would be great. 
Would it be possible to address this in another PR? It might be an opportunity to refactor the constructor for the large test files class a bit.

Reply:
Issue #598 was created to address this issue.